### PR TITLE
Version details page mockup

### PR DESCRIPTION
**Pull request by ruanyl**
### Description
Mockup for model version details page.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/apis/model.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { InnerHttpProvider } from './inner_http_provider';
 export interface ModelSearchItem {
   id: string;
   name: string;
+  description: string;
```
Comment (wanglam) [inline public/apis/model.ts:19]: Is that description represent the version notes? Is that field name confirmed with backend side?

File: public/components/model_version/version_details.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+          <EuiText size="s">
+            {createdTime ? renderTime(createdTime, 'MMM d, yyyy @ HH:mm:ss.SSS') : '-'}
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem style={{ maxWidth: 250 }}>
+          <EuiTitle size="xs">
+            <h4>Last updated</h4>
+          </EuiTitle>
+          <EuiText size="s">
+            {lastUpdatedTime ? renderTime(lastUpdatedTime, 'MMM d, yyyy @ HH:mm:ss.SSS') : '-'}
```
Comment (wanglam) [inline public/components/model_version/version_details.tsx:68]: Could we store all the time format in some place? Seems this time format been used in many place.

Comment (ruanyl) [reply public/components/model_version/version_details.tsx:68]: Sounds good, let me update it.

File: public/apis/model.ts (hunk 1 above)
Comment (ruanyl) [reply public/apis/model.ts:19]: Good question, the description is from the current API response, the new version details API may not have it. I'll leave a TODO item and fix it when integrating the API changes.

**Review by wanglam — APPROVED** [review]

**Pull request merged**